Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabIntegrationManager.cpp

```diff
@@ -1050,5 +1085,233 @@ namespace AzToolsFramework
                 return AZ::EntityId();
             }
         }
+
+        int PrefabIntegrationManager::ExecuteClosePrefabDialog(TemplateId templateId)
+        {
+            auto prefabSaveSelectionDialog = ConstructClosePrefabDialog(templateId);
+
+            int prefabSaveSelection =  prefabSaveSelectionDialog->exec();
+
+            if (prefabSaveSelection == QDialog::Accepted)
+            {
+                SavePrefabsInDialog(prefabSaveSelectionDialog.get());
+            }
+            return prefabSaveSelection;
+        }
+
+        void PrefabIntegrationManager::ExecuteSavePrefabDialog(TemplateId templateId, bool useSaveAllPrefabsPreference)
+        {
+            using namespace AzToolsFramework::Prefab;
```

Comment:
Prefer not to use "using namespace", even inside functions can cause problems.
If you want to bring things is better to do:
using AzToolsFramework::Prefab::MyClass1
using AzToolsFramework::Prefab::MyClass2

Reply:
Removed these. 
Also curious, what kind of problems are you talking about here?

Reply:
An example that @hultonha gave me back then: https://godbolt.org/z/W7dM84rqs
Try uncommenting the using namespace there, and you will see that it doesn't do what you think it would